cmd/cli/app/state/state.go

```diff
@@ -26,14 +26,24 @@ import (
 	"github.com/mitchellh/go-homedir"
 )
 
-func Dump(lr *types.LogRootV1) error {
+type persistedLogRoot struct {
```

Comment:
Would it make sense to make this a map of url -> logroot? instead? So that way we can track multiple at the same time?

Reply:
well if we're going to do that, should we do `map[url][treeSize]*types.LogRootV1` so we can save not just the latest queried state from that server?

Reply:
IMO we probably don't want to store the entire tree state locally for every user - but we do need some way to do that for people that want to run full mirrors.